com.civicrm.hrjobroles/js/src/job-roles/controllers/hr-job-roles-controller.js

```diff
@@ -719,6 +715,22 @@ define([
       }
 
       /**
+       * Extracts, from each job role past and preset, the contact id of every funder
+       *
+       * @return {Array} a list of ids
+       */
+      function extractFundersContactIds() {
+        return _(vm.present_job_roles.concat(vm.past_job_roles))
```

Comment:
They are all [lodash](https://lodash.com/docs/3.10.1#includes) methods, will add a more detailed description in the documentation